sentry-samples/sentry-samples-spring-boot-opentelemetry/src/test/kotlin/io/sentry/systemtest/PersonSystemTest.kt

```diff
@@ -44,4 +44,70 @@ class PersonSystemTest {
                 testHelper.doesTransactionContainSpanWithOp(transaction, "spanCreatedThroughSentryApi")
         }
     }
+
+    @Test
+    fun `create person creates transaction if no sampled flag in sentry-trace header`() {
```

Comment:
TODO check if this test actually fails if `TransactionContext` change is reverted

Reply:
Verified